### PR TITLE
Feat/add group field

### PR DESCRIPTION
**Pull request by GraceRuan**

### CODE REVIEW

File: config/templates/hashicorp_vault/lua/audit.lua [hunk 1]
```diff
@@ -224,11 +224,11 @@ function path_to_service_target(tag, timestamp, record)
...
     if new_record["request"] ~= nil and new_record["request"]["path"] ~= nil then
         local path = new_record["request"]["path"]
-        if string.sub(path, 0, 5) == "apps/" then
-            local path_segment = {}
+        local path_segment = {}
+        if string.sub(path, 0, 5) == "apps/" then 
             for i in string.gmatch(path, "[^/]+") do
                 path_segment[#path_segment + 1] = i
-            end
+            end
```
Comment (mbystedt) [inline config/templates/hashicorp_vault/lua/audit.lua:231]: Remove spaces at end of line

File: config/templates/hashicorp_vault/lua/audit.lua [hunk 2]
```diff
@@ -245,6 +245,16 @@ function path_to_service_target(tag, timestamp, record)
                 code = 2
             end
         end
+        if string.sub(path, 0, 7) == "groups/" then
+            for i in string.gmatch(path, "[^/]+") do
+                path_segment[#path_segment + 1] = i
+            end
+            local groupname = path_segment[3] 
+            if service ~= nil then
```
Comment (mbystedt) [inline config/templates/hashicorp_vault/lua/audit.lua:253]: Will always be nil. group name?

**Review by mbystedt — CHANGES_REQUESTED** [review]

File: config/templates/hashicorp_vault/lua/audit.lua (hunk 2 above)
Comment (GraceRuan) [reply config/templates/hashicorp_vault/lua/audit.lua]: corrected

File: config/templates/hashicorp_vault/lua/audit.lua (hunk 1 above)
Comment (GraceRuan) [reply config/templates/hashicorp_vault/lua/audit.lua]: modified

**Review by mbystedt — APPROVED** [review]
Merge after Dec 31.

**Pull request merged**